connect/runtime/src/main/resources/common/message/ConnectAssignment.json

```diff
@@ -0,0 +1,36 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  // Assignments for each worker are a set of connectors and tasks. These are categorized by
+  // connector ID. A sentinel task ID (CONNECTOR_TASK) is used to indicate the connector itself
+  // (i.e. that the assignment includes responsibility for running the Connector instance in
+  // addition to any tasks it generates).
+  "type": "data",
+  "name": "ConnectAssignment",
+  "validVersions": "0",
+  "fields": [
+    // the error code of this assignment; 0 signals successful assignment
```

Comment:
Why not add the note to the `about` field of the json?

Reply:
oh, I misunderstood your comment. Will copy that.